README.md

```diff
@@ -15,12 +15,9 @@ The goal of this 2-day workshop is to teach _new-to-programming_ data profession
 | Activity   | Start Time | End time |
 |------------|------------|----------|
 | Session #1 | 9:00       | 10:30    |
```

Comment:
Since we are calling the four half-days "sessions", then we need a different name for these - maybe modules?

Reply:
Oh right. Yes modules works well.